### PR TITLE
add requirements.txt

### LINKED ISSUE #27: Requirements file
Additionally to the `Pipfile` we can also generate a `requirements.txt` file for users to install it outside of `pipenv` also potentially with other Python versions.

We can add a make command to generate the `requirements.txt` file from the Pipfile.

### PR DESCRIPTION
**Pull request by maxjeblick**
This PR adds `requirements.txt` file that is generated by `pipenv requirements > requirements.txt`
In future, we can add a CI test that automatically checks if `requirements.txt` is synced with `pipfile.lock`. For now, we would need ensure manually that those files are synced.

Closes #27

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -21,6 +21,10 @@ setup-dev: pipenv
 	$(PIPENV) install --verbose --dev --python $(PYTHON_VERSION)
 	$(PIPENV_PIP) install deps/h2o_wave-nightly-py3-none-manylinux1_x86_64.whl --force-reinstall
 
+.PHONY: export-requirements
+export-requirements: pipenv
+	$(PIPENV) requirements > requirements.txt
```
Comment (psinger) [inline Makefile:26]: Can we somehow add the nightly wave build there?

Comment (pascal-pfeiffer) [reply Makefile:26]: Only very manual, as pipenv can't handle the non-pep8 version.

Comment (pascal-pfeiffer) [reply Makefile:26]: We can add it to the requirements, but not automatically with pipenv.

Comment (maxjeblick) [reply Makefile:26]: Added it manually to `makefile`.

**Comment by pascal-pfeiffer** [comment]
can we make this a GH action somehow?

**Comment by maxjeblick** [comment]
> can we make this a GH action somehow?

Let me check, should be doable

**Review by pascal-pfeiffer — APPROVED** [review]
Thank you @maxjeblick , lgtm!

Having this as a GH action is very helpful.

**Pull request merged**